scripts/install/requestrr.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+# Requestrr installation
+# Author: Brett
+# Copyright (C) 2021 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+#shellcheck source=sources/functions/requestrr
+#shellcheck source=sources/functions/appconfigs
+#shellcheck source=sources/functions/users
+. /etc/swizzin/sources/functions/requestrr
+. /etc/swizzin/sources/functions/appconfigs
+. /etc/swizzin/sources/functions/users
+user=$(_get_master_username)
+_requestrr_download
+
+echo_progress_start "Extracting archive"
+unzip -q /tmp/requestrr.zip -d /opt/ >> "$log" 2>&1
+rm /tmp/requestrr.zip
+mv /opt/requestrr* /opt/requestrr
+echo_progress_done "Archive extracted"
+
+echo_progress_start "Creating requestrr user and setting permssions"
+useradd --system -d /opt/requestrr/ requestrr
+chown -R requestrr:requestrr /opt/requestrr/
+chmod u+x /opt/requestrr/Requestrr.WebApi
+echo_progress_done "Requestrr user has been created & permissions set."
+
+echo_progress_start "Applying Requestrr config..."
+
+echo_progress_start "Grabbing list of apps and integrating with Requestrr"
+_get_sonarr_vars
+_get_radarr_vars
+echo_progress_done "Apps have been added and will be added to Requestrr config."
+
+cat > /opt/requestrr/SettingsTemplate.json << CFG
```

Comment:
This thing is probably gonna change format over time. I think `cat`ing this hardcoded version could prove not very maintainable.

How about we try this `jq` trick? Not sure this one will execute, but it's like `sed -i` but for JSON. There are also alternatives with `sponge` but I'm not sure we need the extra dependency for this.
```bash
cat <<< $(jq '.Authentication.Username = "abcde"' SettingsTemplate.json) > SettingsTemplate.json
```
`jq` is a bit specific with the way it works with bash variables but there are tricks I did for this in the transmission functions already

Reply:
This way we could just keep it to the things we want to change, and not hardcode the whole file here